src/native-stack/views/NativeStackView.tsx

```diff
@@ -216,6 +217,9 @@ const RouteView = ({
     if (fullScreenSwipeEnabled === undefined) {
       fullScreenSwipeEnabled = true;
     }
+    if (fullScreenSwipeShadowEnabled === undefined) {
+      fullScreenSwipeShadowEnabled = true;
```

Comment:
If you do it like this, isn't it `true` by default?

Reply:
I was thinking the same but `fullScreenSwipeEnabled` does it this way and it also is `false` by default. I think it is set elsewhere as `false` so this doesn't matter. But I can change that and maybe `fullScreenSwipeEnabled` also.

Reply:
But the `false` value must be set somewhere right? Better to make this explicit.

Reply:
Maybe it would be better to remove this altogether? Now I see that this is only set when `swipeDirection === 'vertical'` so it doesn't make sense to keep it there if it's not set for other swipe directions. Unless I should move it out of this `if`.

Reply:
I think this prop is not dependent on any other props so we can remove it?